### PR TITLE
design: Add in OSTree delivery format results

### PR DESCRIPTION
**Pull request by dustymabe**
Discussion from:
https://github.com/coreos/fedora-coreos-tracker/issues/23

### CODE REVIEW

File: Design.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+
+### Summary
+
+RPM-OStree supports three delivery models for delivering content to
+end user systems.
+
+- rojig: uses a special rojig RPM and re-assembles OSTree commit from RPMs
+  already on mirrors.
+- OCI: OSTree commits are packaged up in OCI container images and delivered 
+  via a container registry
```
Comment (jlebon) [inline Design.md]: Well... RPM-OSTree doesn't technically support that today, right? Specific platforms can make use of https://github.com/openshift/pivot for now, though there's definitely work to be done before one can do e.g. `rpm-ostree rebase oci://....`.

File: Design.md [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+- OSTree Repo: OSTree commits are stored in an OSTree repo (like a git
```
Comment (jlebon) [inline Design.md]: Minor: I'd put this first, since it's the "standard" option most people are probably familiar with.

Comment (jlebon) [reply Design.md]: Super minor: maybe `OSTree remote` instead?

Everything goes in an OSTree repo in the end :)

File: Design.md (hunk 1 above)
Comment (dustymabe) [reply Design.md]: yeah that is true. I was just trying to give a brief rundown of the options at a high level. suggested edit?

File: Design.md (hunk 2 above)
Comment (dustymabe) [reply Design.md]: I can move it to be first

Comment (dustymabe) [reply Design.md]: I think I'll leave it as OSTree repo if you don't mind :). I added some text to the description to explain the OSTree repo is server side.

File: Design.md (hunk 1 above)
Comment (jlebon) [reply Design.md]: Gotcha. In that case maybe just changing the "RPM-OSTree supports three..." to something like: "There are three proposed delivery models..." ?

Comment (dustymabe) [reply Design.md]: done!

**Pull request merged**